public/src/components/channelManagement/stickyTopBar/stickyTopBar.tsx

```diff
@@ -61,12 +61,14 @@ const useStyles = makeStyles(({ palette, spacing }: Theme) => ({
     },
     marginLeft: spacing(1),
   },
-  buttonText: {
-    fontSize: '14px',
-    fontWeight: 500,
-    textTransform: 'uppercase',
-    letterSpacing: '1px',
+  button: {
```

Comment:
I've moved the color up to the <button> element, rather than the inner <p>, so that the color is automatically overridden when the button is disabled